routes/repo/repo.go

```diff
@@ -235,7 +235,17 @@ func Action(c *context.Context) {
 	case "watch":
 		err = models.WatchRepo(c.User.ID, c.Repo.Repository.ID, true)
 	case "unwatch":
-		err = models.WatchRepo(c.User.ID, c.Repo.Repository.ID, false)
+		if userName := c.Query("user_name"); userName != "" {
+			if c.User.IsAdmin {
+				var user *models.User
+				if user, err = models.GetUserByName(userName); err != nil {
```

Comment:
Make it into two lines with `user, err := ` fits better here.

Reply:
this err used later